tools/check_pod_whitespace_rule

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -ex
+<< 'heredoc_pod_whitespace_rule'
+As a convention the POD and the code needs to have an empty line in between.
+The script checks all the files rather the commit files in the lib directory.
+heredoc_pod_whitespace_rule
+
+
+grep -Pzo "=cut\nsub" -rn lib || exit 0
```

Comment:
Improvement idea: looping through all files one by one and test one by one could provide a better error report to end user also indicating which file is wrong.

Reply:
does it make sense to add this test within the existing https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tools/check_code_style script?

Reply:
> Improvement idea: looping through all files one by one and test one by one could provide a better error report to end user also indicating which file is wrong.

i think you are right about the better reporting however IMO it is a really simple test. you just need to validate that the rule is not violated. the output in the current check will print all the files which do not have the whitespace. So i doubt that the one by one approach will make a big difference. In the other side, it will make the script to run a bit longer.

Reply:
> does it make sense to add this test within the existing https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tools/check_code_style script?

it does. i will do so if it will be requested. Do you want to move the check there?

Reply:
I suggest to actually use `git grep` which can be much faster. This is also what we use in other cases.

Reply:
The thing is that the `git grep` didnt give me the results when i tried. I am not sure why but it doesnt seem to work with those options. Can you check by yourself please? What do i miss?

Reply:
> > does it make sense to add this test within the existing https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tools/check_code_style script?

As i moved the check in the `test-static` i guess there is no anymore a reason to discuss this.

Reply:
I assume your check relies on some special behaviour to grep for the newline. That's fine, just leave it as is